app/HutchManager/Data/ApplicationDbContext.cs

```diff
@@ -23,6 +23,9 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
   public DbSet<ModifierType> ModifierTypes => Set<ModifierType>();
 
   public DbSet<ResultsModifier> ResultsModifier => Set<ResultsModifier>();
+  
+  public DbSet<Agent> Agent => Set<Agent>();
```

Comment:
The Set itself should be plural (`Agents`) just like the table name should be.

The type of record (the Entity) is singular `Agent`.

i.e. one Row of the `Agents` table is an `Agent`.

Since we're here and making migrations anyway, can you also fix `ResultsModifier` in the same way? ;)